standards/request-response.md

```diff
@@ -493,6 +493,36 @@ The usage of non-standard headers is not considered custom headers. For example,
 SPS-CORS-Error: bad origin
 ```
 
+#### SPS-Execution-Context
+
+**Type**: Request
+
+**Support**: OPTIONAL
+
+**Description**: The Execution Context header provides a standard method of indicating the dataflow context that a request should be processed with. This is used to differentiate or test changes in customer configuration of non-production data. This isolates the version of the configuration state used in a request and allows identifying transactions that should leverage a specific tagged or aliased configuration set.
+
+- A request without the presence of this header **MUST** indicate that the execution context is under the `production` dataflow and customer configuration, if applicable.
+- The header value contains any dynamic __string__ value representing a named configuration or tagged dataflow preset.
+    - __preprod__ - Well-known value used to represent non-production customer configuration where named configuration is not supported. Typically used for legacy support. Usage of dynamically named configuration sets is preferred.
```

Comment:
"where named configuration is not supported".. maybe rephrase this a bit. Should say something like "preprod is a well-known value used to represent non-production customer configuration and can be supported as static mode in some legacy services without full dynamic support. Usage of dynamically named configuration sets is preferred.